src/object/srv_obj_migrate.c

```diff
@@ -475,6 +475,59 @@ migrate_pool_tls_lookup_create(struct ds_pool *pool, int version,
 	return tls;
 }
 
+static void
+mrone_recx_daos_vos_internal(struct migrate_one *mrone,
+			     struct daos_oclass_attr *oca,
+			     bool daos2vos, int shard)
+{
+	daos_iod_t *iod;
+	int cell_nr;
+	int stripe_nr;
+	int j;
+	int k;
+
+	D_ASSERT(DAOS_OC_IS_EC(oca));
```

Comment:
D_ASSERT(shard < obj_ec_data_tgt_nr(oca))?

Reply:
ah, ok, will add it later.